ang/civicase/case/list/directives/case-list-table.directive.js

```diff
@@ -203,6 +203,7 @@
       $scope.$bindToRoute({ expr: 'sort.dir', param: 'sd', format: 'raw', default: 'ASC' });
       $scope.$bindToRoute({ expr: 'caseIsFocused', param: 'focus', format: 'bool', default: false });
       $scope.$bindToRoute({ expr: 'viewingCase', param: 'caseId', format: 'raw' });
+      $scope.$bindToRoute({ expr: 'showCasesFromAllStatuses', param: 'sas', format: 'bool' });
```

Comment:
can we rename it to `show_all_statuses` ? I think short names are an issue because it's difficult to determine their purpose, specially when they have a value of `1` or `true`, etc.